src/schema/v1/artwork/index.ts

```diff
@@ -501,6 +502,10 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
       literature: markdown(({ literature }) =>
         literature.replace(/^literature:\s+/i, "")
       ),
+      location: {
+        type: LocationType,
+        resolve: ({ location }) => location,
+      },
```

Comment:
**question**: Are there clients using Metaphysics V1 which will use this new field? If not, we could consider making this a V2-only addition.

Reply:
Not really, it will be used by Pulse, which queries the V2. I was unsure about this, seems fair to make it only a V2 addition